ITISFoundation/osparc-simcore
# 🐛 datcore: fix non-implement get_file in storage

## What do these changes do?
- Implements get_file in storage/datcore_dsm (it was not implemented and never used before)
- Add a new files route in the datcore_adapter and a test for this new route

## Related issue/s
fixes https://github.com/ITISFoundation/osparc-issues/issues/1268

## How to test
Tested in a local deploy:
- Drag and drop a file from the "datcore" storage in the workbench
- Connect the file to an interactive service (e.g. jupyter-math)
- Verify that the file appears in the jupyter-math input folder (this didn't work before this test)

Datcore unit tests:
- They pass when testing against the real pennsieve API with e.g. package `N:package:4a166aa2-858b-46aa-83a3-01485e7e5de7`
## Dev Checklist


## DevOps Checklist
Not applicable
